marklogic-data-hub/src/test/resources/e2e-test/e2eentity.entity.json

```diff
@@ -7,7 +7,6 @@
   "definitions" : {
     "e2eentity" : {
       "required" : [ ],
-      "primaryKey": "id",
```

Comment:
I am trying to check how this primaryKey can make the entity invalid. Will report back.

Reply:
There seems to be an issue with json tests. xml tests are passing ok. I dont think there is anything wrong with the entity having primaryKey.
The error for all the failed MappingE2E tests is:
`Info: Status 500: TDE-INDEX: xdmp:eval("xdmp.documentInsert(\"/8390440750948294347.json\",trace,xdmp.def...", {trace:{trace:{jobId:"10e61e72-45ac-4545-8651-5d4f74bd5880", format:"json", traceId:"8390440750948294347", ...}}}, {database:"15450808607671818307", commit:"auto", update:"true", ...}) -- Error applying template /tde/e2eentity-0.0.1.tdex to document /8390440750948294347.json: TDE-EVALFAILED: Eval for $subject-iri='sem:iri(concat("http://www.marklogic.com/e2eentity-0.0.1/e2eentity/", fn:encode-for-uri(./id)))' returns TDE-BADVALEXPRESSION: Invalid val expression: XDMP-ARGTYPE: (err:XPTY0004) arg1 is not of type xs:string?`

Reply:
This is due to invalid scoping of the tde's generated by ES. Ugh.